LayoutTests/fast/scrolling/mac/async-overscroll-behavior-unscrollable-element.html

```diff
@@ -3,6 +3,7 @@
 <script src="../../../resources/testharness.js"></script>
 <script src="../../../resources/testharnessreport.js"></script>
 <script src="../resources/overscroll-behavior-support.js"></script>
+<script src="../../../resources/ui-helper.js"></script>
```

Comment:
Ditto.

Reply:
`resources/overscroll-behavior-support.js` pulls in UIHelper stuff with all these.